### ISSUE TITLE
Localization not complete in login screen

### ISSUE THREAD

**Issue by lucamosca1** [pop-os/cosmic-greeter#123]
While all the tip for the button in login screen are correctly localized (italian, I assume the login screen is taking the last session language) date is not, and also Auth failure message, when working due to https://github.com/pop-os/cosmic-greeter/issues/122 appear in english.

![Schermata del 2024-09-04 11-45-40](https://github.com/user-attachments/assets/710ac71d-f8f1-491e-b87b-c063c9a77272)

**Comment by lucamosca1** [comment]
Quick update, after OS update (apologise) the date seems correctly localized but in a not correct (from an italian perspective) format. 

![Schermata del 2024-09-04 12-02-28](https://github.com/user-attachments/assets/63f13dce-bbc8-4bd0-a4b9-5b2a2dea703a)

In italy it should be: mercoledì 4 settembre
For date I suppose is just a localization fix, if someone suggest me when to fix I'll check it

Auth failure message still in english.

**Comment by joshuamegnauth54** [comment]
Apparently PAM is supposed to localize the `Password:` prompt, but I'm not sure why it's not working.

**Comment by MinerMax555** [comment]
> Apparently PAM is supposed to localize the `Password:` prompt, but I'm not sure why it's not working.

Actually, in Italian we use the word "Password" too :). It's just borrowed over from English.

**Comment by lucamosca1** [comment]
> > Apparently PAM is supposed to localize the `Password:` prompt, but I'm not sure why it's not working.
> 
> Actually, in Italian we use the word "Password" too :). It's just borrowed over from English.

Yes, the issue is not that string is not localized, is that appears pam_auth instead of Password

**Comment by Etienne-TR** [comment]
In french, `Password:` should be translate into "Mot de passe :".

Also date format is not correct, I get `Samedi, mars 1` instead of `Samedi 1er mars`.

**Comment by Etienne-TR** [comment]
Also if password incorrect the message "Authentication failed" should be translate in "Échec de l'authentification" in case of french localization.

**Comment by mirfilip** [comment]
Using the `cosmic-greeter/noble,now 0.1.0~1744733358~24.04~9e22418 amd64`, when you first log in, it indeed displays the date as it's defined in locale set **but** when you then lock the screen, you end up with with a date formatted (seemingly) with US locale. 

Smells like a bug.
